## Part 1a
1. Line 9 prints:
   
   values added: 20

2. Line 13 prints:
   
   final result: 20

3. Line 9 prints:

    values added: 20

4. Line 13 returns an error because when we use the let keyword, result is only defined in the if statement. Since line 13 is outside of the if statement and is trying to call for result, we get an error since it does not exist. 

5. 